proposals/PROPOSALS.md

```diff
@@ -1,141 +1,126 @@
-* Notify the HFTT on Github using the team label `@haskellfoundation/tech-proposals`. Optionally, the address the HFTT Slack instance or on Discourse (or both!).
+The TWG will address the following concerns:
+ * Soliciting projects from the community, where relevant
+ * Ensuring that relevant stakeholders have been contacted and that their input and needs have been considered
+ * Guiding discussions in a kind and productive direction
+ * Providing a final recommendation as to whether a project should continue
+If the TWG does not reach a consensus, then the working group's decision will be made by majority vote.
+
+Projects undertaken by the Haskell Foundation need not go through this process, and approval of a project by this working group does not guarantee that the Haskell Foundation will actually commit resources. Committee members are tasked with recommending whether a project is likely to be of benefit to the community, not for prioritizing the Foundation's limited resources. However, we expect that, given sufficient resources, recommended projects will be supported by the executive team.
+
+The TWG is a successor to Haskell Foundation Tech Track (HFTT), and will initially consist of its members.
+
+## Membership
+
+The committee will consist of 8 community members and the Haskell Foundation's executive director (ED), or other executive team member.
+Volunteers should be recruited in such a way as to achieve a diversity of technical knowledge and experiences, so as to best serve the entire Haskell community.
+In other words, volunteers should be recruited who have knowledge of and experience with at least the following:
+ * Using Haskell in a variety of environments (research, traditional business applications, free-time open-source work, large teams, small teams)
+ * Using different build tools and dependency management techniques
+ * Domain knowledge such as cryptography, security, performance, and databases
+The ED will report on the current state of the committee's coverage of these areas as membership is updated.
+Applications to join the committee will be decided on by the committee's members.
+Inactive members may be removed by the ED after making a good-faith effort to contact them.
+
+The current members of the TWG are:
+ * Davean Scies (@davean)
+ * Gershom Bazerman (@gbaz)
+ * John Ericson (@Ericson2314)
+ * Hécate (@Kleidukos)
+ * Evie Ciobanu (@eviefp)
+ * David Thrane Christiansen (@david-christiansen)
+
+## Decisions
+
+The committee will make all decisions using a simple majority without secret ballots.
+Committee vote tallies are to be kept confidential, _especially_ those regarding membership.
+In particular, membership vote tallies shall be kept secret from everyone who did not vote.
+Decisions should be made with the participation of 2/3 of the current members of the committee, rounded up.
+The committee may decide to hold regular meetings, or to collaborate asynchronously, or some mix of the two.
```

Comment:
This is flexible which is fine for a charter or whatever, but I'll reiterate that fully async seems unworkable in terms of getting high-quality discussion and feedback.

Reply:
I agree, in practice. I think we should agree to keep having meetings - mostly I want a way to be more flexible if some members are sick or on vacation and can't come to a meeting.